src/js/view/utils/views-manager.js

```diff
@@ -40,6 +40,9 @@ function matchIntersection(entry, group) {
 function scheduleResponsiveRedraw() {
     cancelAnimationFrame(responsiveRepaintRequestId);
     responsiveRepaintRequestId = requestAnimationFrame(function responsiveRepaint() {
+        views.forEach(view => {
+            view.resizeFloatingPlayer();
```

Comment:
Why do we have 4 separate forEach blocks ? @robwalch are you ok with @DanFerrer consolidating them ?

Reply:
@DanFerrer if we call `view.resizeFloatingPlayer` after `view.checkResized` we might be able to get away with using `getSafeRegion` instead of `bounds(_playerElement);` in `view.resizeFloatingPlayer`

Reply:
Using `bounds` will not work properly for outstream players regardless, `getSafeRegion` is patched for outstream to return the correct dimensions so using `getSafeRegion` is the preferred approach here.